com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/VaryingMesh.hlsl

```diff
@@ -198,6 +198,7 @@ FragInputs UnpackVaryingsMeshToFragInputs(PackedVaryingsMeshToPS input)
 struct VaryingsMeshToDS
 {
     float3 positionRWS;
+    float3 positionPredisplacementRWS;
```

Comment:
Don't you miss a #ifdef VARYINGS_NEED_POSITIONPREDISPLACEMENT_WS here?

Reply:
I removed this, so it's not needed in the struct at all, outside of Shader Graph.